packages/api/src/api/expoJudgingSession/id/results/index.ts

```diff
@@ -0,0 +1,6 @@
+import { Router } from 'express';
+import { get } from './get';
+
+export const results = Router();
```

Comment:
Here's how I know you didn't test this method 😉 
```suggestion
export const results = Router({ mergeParams: true });
```